### PR TITLE
Retrieve node information/network status from v5 - Closes #235

### LINKED ISSUE #235: Retrieve node information/network status from v5
### Description

Make sure that basic information like `lastBlockId` and `height` are available through the lisk-elements library.

### Motivation

- Retrieval of basic information of the core node is essential for the whole 

### Acceptance Criteria

- The main goal is to establish connection with Lisk Core
- lisk-elements library needs to be used

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #235 

### How was it solved?

- [x] Initiate SDKv5 using rpc call
- [x] Establish connection with Lisk Core
- [x] lisk-elements component (lisk-api-client) used
- [x] Retrieve node/info from lisk-core
- [x] Includes `lastBlockId` and `height`

### How was it tested?
Local + Jenkins

### CODE REVIEW

File: services/core/shared/core/compat/common/constants.js [hunk 1]
```diff
@@ -23,11 +24,19 @@ let coreVersion = '1.0.0-alpha.0';
...
 const getNetworkConstants = async () => {
-	// const expireMiliseconds = Number(config.ttl.stable) * 1000;
-	// const result = await coreApiCached.getNetworkConstants(null, { expireMiliseconds });
-	const result = await http.get('/node/constants'); // Necessary to remove cyclic dependency
-	if (!isProperObject(result)) return {};
-	return result;
+	try {
+		let result = await http.get('/node/constants'); // Necessary to remove cyclic dependency
+		if (Object.getOwnPropertyNames(result).length === 0) {
+			const apiClient = await ws.getClient();
```
Comment (MichalTuleja) [inline services/core/shared/core/compat/common/constants.js:30]: Is the node URL taken from the config?

Comment (nagdahimanshu) [reply services/core/shared/core/compat/common/constants.js:30]: Yes, node URL is taken from config file and file `services/core/shared/core/compat/common/wsRequest.js` is responsible for it.
```
const { createWSClient } = require('@liskhq/lisk-api-client');
const config = require('../../../../config');

const liskAddress = config.endpoints.liskWs;
let clientCache;

const getClient = async () => {
    try {
        clientCache = await createWSClient(`${liskAddress}/ws`);
        return clientCache;
    } catch (err) {
        return {
            data: { error: 'Action not supported' },
            status: 'METHOD_NOT_ALLOWED',
        };
    }
};```

File: services/core/shared/core/compat/sdk_v5/network.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const coreApi = require('./coreApi');
+
+const getNetworkStatus = async () => {
+	const core = await coreApi();
+	const status = await core.getNetworkStatus();
```
Comment (sameersubudhi) [inline services/core/shared/core/compat/sdk_v5/network.js:20]: This line of code isn't necessary with the currently recommended change in `services/core/shared/core/compat/sdk_v5/coreApi.js`.
```suggestion
const coreApi = require('./coreApi');

const getNetworkStatus = async () => {
	const status = await coreApi.getNetworkStatus();
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Please consider the review comments.

File: services/core/shared/core/compat/sdk_v5/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { getNetworkStatus } = require('./network');
+
+module.exports = {
+    ...require('../sdk_v4'),
```
[the remark this answers is not in the document]
Comment (nagdahimanshu) [reply services/core/shared/core/compat/sdk_v5/index.js:19]: The only reason to require `sdk_v4` functions is to build `sdk_v5` properly. Else, while building `sdk_v5`, it throws errors message given functions does not exists and stop the process. 
While `sdk_v5` development, we can replace it.

File: services/core/shared/core/compat/common/wsRequest.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ */
+const { createWSClient } = require('@liskhq/lisk-api-client');
+const config = require('../../../../config');
+
+const liskAddress = config.endpoints.liskWs;
+let clientCache;
+
+const getClient = async () => {
+    try {
+        clientCache = await createWSClient(`${liskAddress}/ws`);
```
[the remark this answers is not in the document]
Comment (nagdahimanshu) [reply services/core/shared/core/compat/common/wsRequest.js]: Done!

File: services/core/shared/core/compat/sdk_v5/coreApi.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        const result = await clientCache.node.getNodeInfo();
+        return { data: result };
+    };
+
+    return {
+        getNetworkStatus,
+    };
+};
+
+module.exports = coreApi;
```
[the remark this answers is not in the document]
Comment (nagdahimanshu) [reply services/core/shared/core/compat/sdk_v5/coreApi.js]: Done!

File: services/core/shared/core/compat/sdk_v5/network.js (hunk 2 above)
Comment (nagdahimanshu) [reply services/core/shared/core/compat/sdk_v5/network.js]: Done!

File: services/core/shared/core/compat/sdk_v5/coreApi.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { getClient } = require('../common/wsRequest');
+
+const getApiClient = async () => {
+    const clientCache = await getClient();
+    return clientCache;
```
Comment (sameersubudhi) [inline services/core/shared/core/compat/sdk_v5/coreApi.js:20]:
```suggestion
    const wsClient = await getClient();
    return wsClient;
```

**Review by sameersubudhi — APPROVED** [review]
Looks good! Have shared a minor suggestion regarding a variable name.

**Pull request merged**